pyproject.toml

```
@@ -125,7 +125,8 @@ line-length = 88
 src = ["earthaccess", "stubs", "tests"]
 
 [tool.ruff.lint]
-extend-select = ["I", "T20"]
+extend-select = ["I", "T20", "D"]
+ignore = ["D100", "D101", "D102", "D103", "D104", "D105", "D107", "D205", "D401", "D417", "I001"]
```

```suggestion
ignore = ["D1", "D205", "D401", "D417"]
```

I think we can ignore all `D1*` errors like this :point_up: Can you check if this works?

We need to keep `I001` on and fix any unsorted import errors. Did the autofixer not take care of it? If this one needs to be manually fixed, we need to do it :)